samples/cellular/location/overlay-wifi-scan-only.conf

```diff
@@ -4,7 +4,10 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
-# Overlay to use nRF7002 EK on top of nrf9160 DK for Wi-Fi scanning
+# Overlay to use  Wi-Fi scanning
```

Comment:
Is a line break needed?

Reply:
Can we skip this kind of nitpicking in the future?